packages/metro-config/package.json

```diff
@@ -17,10 +17,10 @@
     "jest-validate": "^24.9.0",
```

Comment:
`jest-validate` 24.9 also has `pretty-format` in the runtime dependency list. Do we need to make this same change to their project?